src/engine/currencyEngine.js

```diff
@@ -108,7 +108,10 @@ export class CurrencyEngine {
         ])
       },
       onAddressesChecked: this.abcCurrencyEngineOptions.callbacks
-        .onAddressesChecked
+        .onAddressesChecked,
+      onTxidsChanged: (txids: Array<string>) => {
```

Comment:
This can be just:
`onTxidsChanged: this.abcCurrencyEngineOptions.callbacks.onTxidsChanged`

Since both have the same function signature, there is no need to create a closer

Reply:
got it